src/components/App/App.spec.js

```diff
@@ -1,21 +1,20 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import App from './App';
-import Calculator from '../Calculator/Calculator';
+
+jest.mock('../Calculator/Calculator', () => 'Calculator')
 
 describe('App', () => {
-    let wrapper;
+    let subject;
     beforeEach(() => {
-        wrapper = shallow(<App />);
+        subject = shallow(<App />);
     })
 
-    it ('should render correctly', () => expect(wrapper).toMatchSnapshot())
-    
-    it ('should render a div', () => {
-        expect(wrapper.find('div').length).toEqual(1);
+    it ('renders a div with className "app-container"', () => {
+        expect(subject.find('div').length).toEqual(1);
```

Comment:
```suggestion
        expect(subject.find('div').hasClass("app-container")).toBe(true);
```

Like we've talked about, let's finite our test to check for something distinctive 👍

Reply:
```suggestion
        expect(subject.find('div.app-container').length).toBe(1);
```
What do you think about just adding the class to the selector?

Reply:
Absolutely, same results just a little more concise 👍